gstd/src/util.rs

```diff
@@ -90,3 +89,22 @@ pub fn to_leak_ptr(bytes: impl Into<Vec<u8>>) -> *mut [i32; 2] {
     core::mem::forget(bytes);
     ptr
 }
+
+#[cfg(feature = "log")]
+pub(crate) fn u32_with_sep(num: u32) -> String {
+    u64_with_sep(num.into())
+}
+
+#[cfg(feature = "log")]
+pub(crate) fn u64_with_sep(num: u64) -> String {
+    use crate::prelude::ToString;
+
+    num.to_string()
+        .as_bytes()
+        .rchunks(3)
+        .rev()
+        .map(core::str::from_utf8)
+        .collect::<Result<Vec<&str>, _>>()
+        .unwrap()
+        .join(",")
```

Comment:
Maybe underscore `_` is better as a separator (to be Rust-like)? The comma may be confused with a decimal point.